parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -285,7 +285,15 @@ if $REBOOTREQUIRED; then
 else
     if [[ $OS == $UBUNTU_OS_NAME ]]; then
         # logs_to_events should not be run on & commands
-        /usr/lib/apt/apt.systemd.daily &
+        {{- if EnableUnattendedUpgrade }}
+        systemctl unmask apt-daily.service apt-daily-upgrade.service
+        systemctl enable apt-daily.service apt-daily-upgrade.service
+        systemctl enable apt-daily.timer apt-daily-upgrade.timer
+        {{- end }}
+        # this is the DOWNLOAD service
+        # meaning we are wasting IO without even triggering an upgrade 
+        # -________________-
+        systemctl restart apt-daily.service
```

Comment:
I changed this to invoke the oneshot service directly since backgrounded task is pretty useless and would exit on cse handler termination without nohup or daemonize. 

but the timer itself will execute it nearly immediately, not sure we should bother with this.

Reply:
@paulgmiller for thoughts?